ui/src/app/pages/data/data-set/export-dataset-modal.spec.tsx

```diff
@@ -62,16 +70,22 @@ describe('ExportDatasetModal', () => {
       expect(screen.queryByLabelText('Please Wait')).toBeNull();
     });
 
-  const changeNotebookName = (newNotebookName) => {
-    fireEvent.change(findNotebookNameInput(), {
-      target: { value: newNotebookName },
-    });
+  const changeNotebookName = async (newNotebookName) => {
+    await user.type(findNotebookNameInput(), newNotebookName);
   };
 
   const clickExportButton = () => {
     fireEvent.click(findExportButton());
   };
 
+  const clickShowCodePreviewButton = () => {
```

Comment:
naming: show vs. see?

Reply:
Updated to see in order to be consistent with button text.